src/webapp/components/DeliveryTable.js

```diff
@@ -92,7 +92,7 @@ const DeliveryTable = ({ rows }) => {
         </TableHead>
         <TableBody>
           {formattedRows.map((row) => (
-            <DeliveryTableRow row={row} key={row.code} />
+            <DeliveryTableRow row={row} key={row.Code} />
```

Comment:
This will fix the console warnings about a lack of unique keys :) (`code` isn't a field, but `Code` is)